mrtt-ui/package.json

```diff
@@ -34,9 +34,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix",
-    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
+    "lint": "eslint ./src/**/*.{js,jsx,tsx,ts}",
+    "lint:fix": "eslint './src/**/*.{js,jsx,tsx,ts}' --fix",
+    "prettier": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
```

Comment:
I changed this from being 'format' which feels like it could mean many things. I dont really care too much though, if its important to someone that it stays that way, I can change it back.

Reply:
i'm ok with renaming it